.github/workflows/main.yml

```diff
@@ -258,7 +267,7 @@ jobs:
         uses: actions/upload-artifact@v3
         if: matrix.packager && startsWith(github.ref, 'refs/tags/') != 'true'
         with:
-          name: ${{ matrix.cross_os || runner.os }} binaries
+          name: Drawpile-${{ matrix.cross_os || runner.os }}-Qt${{ matrix.qt }}
```

Comment:
Putting Drawpile- in front of it just makes it easier to find the file when it gets dumped into a downloads directory, putting the Qt version after it is there so that it's clear which job it belongs to, since it's not immediately obvious that only the Qt5 jobs create release artifacts.